rviz_common/include/rviz_common/render_panel.hpp

```diff
@@ -105,7 +105,6 @@ class RVIZ_COMMON_PUBLIC RenderPanel : public QWidget
   static const Ogre::Vector3 default_camera_pose_;
 
 // TODO(wjwwood): reenable these and pass down to rviz_rendering::RenderWindow
```

Comment:
I'm not sure what passing down to the render window implies.

Reply:
I don't think it's a good idea to pass this down to the render window. It has nothing to do with Ogre and rendering, so I don't think this should be in `rviz_rendering`. I think this comment can just be ignored and deleted.

However, the separation of concerns between the `rviz_rendering::RenderWindow` and the `rviz_common::RenderPanel` has never been properly finished (Ogre is still leaking too much into `rviz_common`), so maybe William had more in mind here...

Reply:
@wjwwood Let me know if I can remove this TODO (or update it accordingly).

Reply:
I don't remember, but I do remember that it might be required to pass information down to ogre so you could do things like cause 3d objects to be "highlighted" when clicked on (for example). But I don't have a concrete use case. If you didn't need it for this work, then you can likely remove this todo.

Reply:
I wasn't sure how to test this other than mouse around on a blank view (ie. confirming that the fallback to move tool works). I will leave this PR open until I confirm it works with interactive markers  (#98).

Reply:
Seems to work as expected with what I've got so far for the interactive marker display (https://github.com/ros2/rviz/tree/jacob/interactive_markers).
Removed TODOs in 69ee286.